fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/cache/LRUCache.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.cache;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An LRU implementation which limits the maximum number
+ * of entries and ejects older entries. Null values are
+ * not permitted in this implementation.
+ */
+public class LRUCache<K,V> extends LinkedHashMap<K,V> {
```

Comment:
should this get reconciled with `com.ibm.fhir.core.util.LRUCache`?  Its basically the same approach, although this has a tad more.

Reply:
I think this should become a backlog issue.

Reply:
creating an issue

Reply:
#1643